### ISSUE TITLE
ETC Core Devs Call 22: Proposed Rejection of ECIP-1049

### ISSUE THREAD

**Issue by gitr0n1n** [ethereumclassic/ECIPs#460]
Agenda:
https://ethereumclassic.org/blog/2022-02-21-core-devs-call-22-ecip-1049-proposed-rejection
Discuss fate of ECIP-1049 after three years. (ECIP-1000 clause).

Focus: REJECT Keccak-256 Mining Algorithm Change due to a high-probability risk of Contentious Chain Split between GPU Miners on ETCHash and FPGA & ASIC Miners on Keccak-256. ECIP-1049 is in violation of Ethereum Classic founding documents and the ECIP process. At this point, the contentious proposal has negative externalities on the network and is a resource drain. Move to reject the proposal after three years of technical discussion. If the proposal is not rejected, begin plans for a chain split.

Decision to be made:
+ Move to `Rejected` status, or;
+ Set a Block and Push Contentious Fork with a `Chain Split`

ECIP-1000 Clause: "ECIPs should be changed from Draft or Last Call status, to Rejected, upon request by any person, if they have not made progress in three years. Such a ECIP may be changed to Draft status if the champion provides revisions that meaningfully address public criticism of the proposal, or to Last Call if it meets the criteria required as described in the previous paragraph."

The reason for "Rejected" status under this clause is that the champion (Alex Tsankov) has not met this requirement during the three years: "the champion provides revisions that meaningfully address public criticism of the proposal". Rather, the champion (Alex Tsankov) has ignored much valid criticism and abandoned the proposal. https://ecips.ethereumclassic.org/ECIPs/ecip-1000

Follow Up from: https://github.com/ethereumclassic/ECIPs/issues/382
Formal Proposed Rejection: https://github.com/ethereumclassic/ECIPs/issues/394#issuecomment-1022909537
Documented Github Opposition: https://github.com/ethereumclassic/ECIPs/issues/394#issuecomment-828160552

If time permits: review ECIP-1094 and ECIP-1096 for activity. Newer proposals but appear to be abandoned by the authors as well. Should these be `Withdrawn`?

Please review the issue thread to find the most up to date information.
Related Discussions:
ECIP-1049: Change the ETC Proof of Work Algorithm to Keccak256 #8
ECIP-1049: Change the ETC Proof of Work Algorithm to Keccak-256 #13
ETC Core Devs Call(s) 2020 Q3: Hardfork #333
ECIP-1095: Change ETC PoW to "vanilla" Sha-3 Discussion #342
ETC Core Devs Call 13 & 14 #362
ETC Core Devs Call 15 - ECIP-1049 Breakout Session Keccak-256 #382
SHA3 Precompile ethereum/EIPs#2951
Core Devs Call 15 Recording https://vimeo.com/464336957
Change the ETC Proof of Work Algorithm to Keccak-256 #394
Admin Clean Up on ecip-1049: https://github.com/ethereumclassic/ECIPs/pull/400
Core Devs Call 19 Recording https://www.youtube.com/watch?v=WySNxZbDEkQ
Community Call 005 Recording https://youtu.be/HaDANZN-ZUU?t=1585
Community Call 010 Recording https://youtu.be/6DRZEaKkpb4?t=3411
Community Call 011 Recording https://www.youtube.com/watch?v=ad_grFagA5k
Community Call 012 Recording https://youtu.be/GCBv1VCN2tE?t=3339
Community Call 013 Recording https://www.youtube.com/watch?v=HQ9IKu3PVkA
ETC Core Devs Call 22: Proposed Rejection of ECIP-1049 #460
Pull Request- Rejected Status ecip-1049: https://github.com/ethereumclassic/ECIPs/issues/465

It should be noted in this new discussion thread, this ECIP appears to be contentious (as documented in all the previous threads/recordings) and has a high-probability of a chain split between the GPU Miners on ETCHash and the FPGA & ASIC miners on Keccak-256.

![Etchash-vs-Keccak256](https://user-images.githubusercontent.com/58588775/154829756-4ef2ca53-d926-4514-949d-feee686c4c2a.png)

How to join:
Topic: ETC Core Devs Call 22: Proposed Rejection of ECIP-1049
Time: February 21, 2022
Time: 17:00 UTC

Meeting Link: https://ethereumclassic.org/discord
Channel: community-calls
@ethereumclassic/all-hands

**Comment by gitr0n1n** [comment]
Formal Proposed Rejection: https://github.com/ethereumclassic/ECIPs/issues/394#issuecomment-1022909537

Documented Github Opposition: https://github.com/ethereumclassic/ECIPs/issues/394#issuecomment-828160552

**Comment by gitr0n1n** [comment]
"So the Implementation section of ECIP-1049 doesn't have that level of clarity (this one correct? ethereumclassic/ECIPs#13). It seems to be an extension of the rational section. That is perhaps my biggest beef in that the spec doesn't in fact specify anything with clarity. It has some hints but is missing actual details, such as what is H sub 'n bar'?

There needs to be a "specification" section in ECIP-1049, as noted by the ecip template. These three bullet points should be in that section. There also needs to be clear specification as to the proposed difficulty bump. And some justification of the 100x difficulty bump, it appears to be pulled from thin air.

Also, there are so many versions of ECIP floating around I think the PR description and ultimate merge commit needs to reference the "official" location of the ECIP it's implmementing."

https://github.com/hyperledger/besu/pull/1882#discussion_r573417569

**Comment by gitr0n1n** [comment]
"I'm going to be a bit of a hard nose about this, but until the referenced ECIP (https://ecips.ethereumclassic.org/ECIPs/ecip-1049) specifies what has been discussed in this PR I'm not going to give it a serious review. If we are implementing what should be specified in an ECIP then the ECIP should actually specify something.

I'm not looking for yellow paper equations but at least "this field now has this data (xxxx)" "this field now has this data (yyy)" "to calculate that value do XYZ." "If I J and K then the block is valid, otherwise it is invalid."
https://github.com/hyperledger/besu/pull/1882#issuecomment-784499616

**Comment by bobsummerwill** [comment]
IHMO this meeting should not occur until after the Mystique hard-fork.
I would suggest two weeks later than the current suggested date.

On a personal note, I will be unable to attend any meeting at this time-slot because it falls at 7am for me and I have parental responsibilities at that time of day.  Two hours later is workable.  Mondays and Wednesdays are the best days.

**Comment by bobsummerwill** [comment]
I would argue against the premise of this meeting, and the obvious bias in the two "decision to be made" choices.
There is an obvious third choice, which is to leave the ECIP in Draft status, which I would argue is entirely appropriate for the current state of the proposal.

The assertion that there has been "no progress" on the proposal within the last three years is demonstrably false.

* A PoC on Besu was implemented by Whiteblock
* Production quality support in Besu was implemented in Besu by @atoulme, together with other rework and cleanup to the mining support in Besu generally.   Vanilla Besu today includes support for Keccak256 mining and the Astor testnet.
* Support for Keccak256 in Core-Geth was implemented and tested, though not integrated yet.
* Support for Keccak256 in mining pool software was implemented.
* Henry Quan wrote a whitepaper and blog post analyzing mining options and potential transition landscape.

The cherry picked quotes above are a year old and were part of the review process which ended in the changes being merged into Besu.   They were not show-stoppers.

**Comment by bobsummerwill** [comment]
With regard to the bullet-points in the Formal Proposed Rejection:

* https://github.com/ethereumclassic/ECIPs/issues/394#issuecomment-1022909537

One point which I would agree with is that a new champion is required, and I would be happy to step into that role myself.

The ECIP text itself should also be updated to reflect the work which has been done since 2019, and to note its own issues, shortcomings and remaining TODOs.   As a Draft proposal I would be happy to agree that it is still a work-in-progress.  That is no reason to reject it, especially given the work which has gone into implementation and testing and given the broad interest in the proposal (on both sides of the fence).

All of the other bullet-points I would argue are fairly subjective.

**Comment by bobsummerwill** [comment]
Sorry - trying unsuccessfully to add @atoulme as an assignee, because he is an ECIP Editor.

**Comment by bobsummerwill** [comment]
Another snippet from ECIP-1000:

"The ECIP editor will not unreasonably reject an ECIP."

"Reasons for rejecting ECIPs include duplication of effort, disregard for formatting rules, being too unfocused or too broad, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the Ethereum Classic philosophy."

I would argue that this Rejection Proposal **is** would be an unreasonable rejection.   There is support for the proposal.   Substantial work has been done around implementation, testing and exploration.  Shortcomings in the specification itself can be addressed.

**Comment by gitr0n1n** [comment]
> IHMO this meeting should not occur until after the Mystique hard-fork.
> I would suggest two weeks later than the current suggested date.

I'm open to that date: February 21, 2022 at 15:00 UTC

**Comment by gitr0n1n** [comment]
> One point which I would agree with is that a new champion is required, and I would be happy to step into that role myself.

This is one of the cited reason to hold this meeting as it has been abandoned by the author.


> I would argue against the premise of this meeting, and the obvious bias in the two "decision to be made" choices.
> There is a simple third choice, which is to leave the ECIP in Draft status, which I would argue is entirely appropriate for the current state of the proposal.

The reason that these two options were cited is because there is a vocal opposition that has committed support for the GPU side of the chain. Given that this is not a hashing war, that asserts this proposal as 100% a chain split event after 3 years of attempted consensus building. The consensus simply is not there. So on this call we can talk about how you want to approach that and the only real two options are: Reject/Withdraw or push this proposal on the network and trigger a chain split. To not acknowledge this is to not acknowledge the three years of discussion around this topic.

This will be a pragmatic conversation like the CDC 15 was on this topic. I expect the topic to be long and drawn out. But at the end of the day- that is the decision on the table after three years of contentious discussion. You need to vocalize on video if it is your intent to ignore the committed opposition to the Etchash and GPU miners and push this proposal.

**Comment by gitr0n1n** [comment]
> Another snippet from ECIP-1000:
> 
> "The ECIP editor will not unreasonably reject an ECIP."
> 
> "Reasons for rejecting ECIPs include duplication of effort, disregard for formatting rules, being too unfocused or too broad, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the Ethereum Classic philosophy."
> 


Aside from all the other cited issues in the formal proposal to `Reject` ECIP-1049, on this specific bullet 1049 could be rejected via:
+ disregard for formatting rules
+ being too unfocused or too broad
+ being technically unsound
+ not providing proper motivation
+ not in keeping with the Ethereum Classic philosophy

But there are many more reasons cited:
https://github.com/ethereumclassic/ECIPs/issues/394#issuecomment-1022909537

For these reasons, I believe it important we have this conversation in a timely manner and come to a decisive conclusion. Especially given the recent tactics of the ECIP-1049 proponents to attack Ethereum Classic's social layer via using sock puppets to gaslight, fearmonger, disinform, FUD, paint doomsday scenarios, and hijack other productive agenda items. A clear decision on this will end the consensus debate that is gridlocked.

**Comment by gitr0n1n** [comment]
> The ECIP text itself should also be updated to reflect the work which has been done since 2019, and to note its own issues, shortcomings and remaining TODOs. As a Draft proposal I would be happy to agree that it is still a work-in-progress. That is no reason to reject it, especially given the work which has gone into implementation and testing and given the broad interest in the proposal (on both sides of the fence).

I believe this is an admission by you that this proposal is `WIP` at best. I agree with this sentiment because it has been redrafted two or three times already and we also had the Vanilla SHA3 proposal die. This would be the fourth of fifth draft of the proposal in three years. This proposal simply is not ready in the slightest for a $3B network to consider- which is why the opposition to the porposal has suggested numerous times that the proponents should fire up this idea on a brand new EVM to develop the idea. If this idea was in a more mature state, it may be applicatable for the ECIP process- but instead this is a resource drain on the network- a network with scarce resources. For these reasons, the proponents of ECIP-1049 should really take a look in the mirror and understand the damage they have caused this network over the past three years by ignoring many steps in the ECIP process to push this proposal on the network at all costs.

I look forward to the pragmatic and honest conversation during CDC 22.

![6305ni](https://user-images.githubusercontent.com/58588775/151633094-6b8d5b62-3194-4431-9fad-c23efb98b0a9.png)

**Comment by gitr0n1n** [comment]
1.5 years ago strong opposition still rings true to this proposal. The authors are not addressing these concerns and some are addressable because its frankly a bad idea. and one that is not compatible with this 6 year old network.

From @Dexaran :
https://github.com/ethereumclassic/ECIPs/issues/342#issuecomment-685208360
https://github.com/ethereumclassic/ECIPs/issues/342#issuecomment-685840814
https://github.com/ethereumclassic/ECIPs/issues/342#issuecomment-687640021

**Comment by gitr0n1n** [comment]
From @iquidus :
https://github.com/ethereumclassic/ECIPs/issues/13#issuecomment-701325057
https://github.com/ethereumclassic/ECIPs/issues/13#issuecomment-701468913
https://github.com/ethereumclassic/ECIPs/issues/13#issuecomment-701570314
https://github.com/ethereumclassic/ECIPs/issues/13#issuecomment-701648711

**Comment by gitr0n1n** [comment]
From @Dexaran :
https://github.com/ethereumclassic/ECIPs/issues/342#issuecomment-685208360
https://github.com/ethereumclassic/ECIPs/issues/342#issuecomment-685840814
https://github.com/ethereumclassic/ECIPs/issues/342#issuecomment-687640021

**Comment by wpwrak** [comment]
A transition plan would also have to include an analysis of the safety of the consensus process at all stages of the transition. This analysis should not only consider technical issues immediately tied to protocol changes, but also the impact on the ETC ecosystem, in particular where it relates to mining.

Specifically, a rapid change to SHA3 would reset the ETC mining ecosystem to a state where only CPUs, GPUs, and FPGAs would be able to participate in the PoW process, but it is generally assumed that ASICs would not become available for a significant amount of time (at least for several months, maybe more than a year). While Ethash/ETChash maintains a relatively level playing field between GPUs and ASICs, SHA3 would have a very steep technology gradient, with large performance differences between CPUs, GPUs, FPGAs, and ASICs. Also, ASICs at different technology nodes would show large performance differences.

In most of the discussion about SHA3, the present "stable state" of the ETC ecosystem with ETChash was compared to a "stable state" as it is found in BTC mining, where mining technology has fully matured, and the mining ecosystem has grown as part of the overall BTC economy. A transition process to SHA3 would have a period where technology gaps would still have to be filled, and major new investment in mining equipment would be required. A "stable state" would therefore not exist until the (technological and economical) transition process has completed.

An analysis should quantify the expected performance of miners using different technologies, both in terms of hashes per equipment cost (e.g., some ETChash equipment is in the 100-200 kH/s per USD range, some BTC (SHA256) equipment is in the 5-10 GH/s per USD range.), as in terms of hashes per energy (e.g., some ETChash miners operate at around 1 MH/J, some BTC miners operate at around 30 GH/J), for calculating the hashrate that can be expected, based on the initial GPU miner population, possible FPGA mining, and then ASICs, considering projected investment into mining equipment. (Assuming here that CPU mining will be insignificant; also, the numbers in this paragraph are picked somewhat arbitrarily, and are for illustration only.)

The inrush of Ethash/ETChash hashrate from miners migrating to ETC after the ETH "merge" is likely to upset ETC mining profitability, for at least some time. SHA3 proponents have claimed that miners threatened with no longer being able to mine profitably (i.e., an excess of available hashrate would squeeze them out of the mining ecosystem) are likely to conspire to perform coordinated attacks against ETC. A transition to SHA3 would also create numerous situations where similar conditions would exist, e.g.,
- before a transition, for ASIC miners, which are not able to transition to SHA3;
- for GPU miners, if a significant amount of "cheap" FPGA hashrate may squeeze them out of the ecosystem;
- for GPU and FPGA miners, if and when SHA3 ASICs are developed, and squeeze both out of the ecosystem;
- possibly for early ASICs miners, if substantially more efficient ASICs are developed, and squeeze them out of the ecosystem.

Also, if a transition to SHA3 were to happen before the ETH "merge", and ETC would thus be in the phase where GPUs provide most of the (SHA3) hashrate, the massive increase of available GPU-based hashrate following the "merge" would affect SHA3-based ETC mining in similar ways as the ETChash-based mining will be affected if ETC is still using ETChash. This risk is in addition to the risks caused by the performance leaps during the "ramp-up" process.

**Comment by gitr0n1n** [comment]
I am asking ETC Coop to draft a document and post it to all socials acknowledging that SHA3/ECIP-1049 is `contentious` in nature and all previous articles published by the proponents citing `SHA3` as a sure thing for ETC were inaccurate. This is to repair the damage the misinformation campaign has had on the network and its relation with GPU miners/participants.

These articles were extremely disingenuous to the retail users on ETC, GPU miners, and those following from the peripheral. As many read those articles and thought "oh so this is already planned for activation".

If needed I'll crawl back and pull up these tweets and articles to illustrate the misinformation campaign you guys executed. 

Those are the actions of `Bad Actors`, if you're not a bad actor then a correction article would be appreciated and a link to the CDC 15 video from October 2020 with a mention of the Rejection proposal on February 21, 2022 17:00 UTC.
cc: @ethereumclassic/all-hands 

Proof of misleading effects of the misinformation campaigns and paid articles: https://duckduckgo.com/?q=ethereu+classic+sha3

Notice ETC Coop's involvement: 
Pinned on their page: https://medium.com/etccooperative/why-change-the-proof-of-work-algorithm-to-keccak-256-sha3-e327b8313824

![Bob-required-engagement](https://user-images.githubusercontent.com/58588775/151719281-e6085bb3-be95-454b-9ed9-6fb116e20654.png)

**Comment by bobsummerwill** [comment]
So I can make 17:00 ETC on 21st February.

**Comment by stevanlohja** [comment]
@gitr0n1n I think it's great that you want to organize a meeting, but this title, content, and thread are very loaded with bias and as @bobsummerwill has mentioned there's been substantial work around the proposal from implementations, testnets, mining clients, education, and community support. You in particular have had a lot of engagement from Sha3 folks in our community for the past week. I'm assuming your coordination with the ECIP author is an after-the-fact consideration here to. Nurture your privilege as an ECIP Editor more professionally.
cc ECIP Editor: @meowsbits @IstoraMandiri @q9f

**Comment by gitr0n1n** [comment]
@stevanlohja Please refrain from censoring the ECIP process with your personal agenda. We have seen you executing social attacks via sock puppets on Discord, posting misleading information on the Ethereum Classic twitter and denigrating the network everywhere. You are acting and have been acting in `bad faith` at every step of this proposal. Closing this issue is just another example of these malicious actions as a proponent that has misinformed the general public on the state of the proposal and ECIP-1095.

Please think about how you are violating the network at every step of the way to push your single agenda.

**Comment by gitr0n1n** [comment]
@ethereumclassic/ecip-editors can we do an audit of @stevanlohja permissions? I believe he stepped down as an ECIP editor and should not have the authority to maintain this repo, especially given his recent unhinged malicious actions.

We saw a similar need when Wei abused his positions to push his singular point of view on the network. We likely should review @stevanlohja 's permission across the board and monitor his actions moving forward to see if intervention is needed.

**Comment by gitr0n1n** [comment]
From @realcodywburns :

> DontPanicBurns | Chippr Bots — 01/30/2022
> i was shaddow tagged somewhere above. My official position is that sha3 has been an utter failure by all measures. Iohk was tricked into wasting development effort on sha3 and failed to develop their client. There is nothing to gain by switching. It is a huge risk with zero gain. The topic itself is brought up by nontechnical blowhards whenever any other progress is being discussed and it is exhausting.  ignore this proposal and anyone who supports it, they are an enemy of etc. I hope this makes my position clear. lfg

![Opposition](https://user-images.githubusercontent.com/58588775/152251713-e962f641-eb2c-4ff3-9ad7-1b6848615d7b.png)

**Comment by bobsummerwill** [comment]
Just to reiterate that I cannot make the meeting at its current time of 17:00 UTC.

19.00 UTC works for me.

**Comment by ETCDAOorg** [comment]
> Just to reiterate that I cannot make the meeting at its current time of 17:00 UTC.
> 
> 19.00 UTC works for me.

Bob, I previously adjusted the time from 15:00 UTC to 17:00 UTC per your request.
https://github.com/ethereumclassic/ECIPs/issues/460#issuecomment-1024583421

To verify, Now saying 17:00 UTC does not work for you and 19:00 UTC would be a better time?

Please confirm the desired time one last time on your end prior to changing the meeting time. I'd like to make sure this is the last change, if possible. Thanks.

**Comment by gitr0n1n** [comment]
> Just to reiterate that I cannot make the meeting at its current time of 17:00 UTC.
> 
> 19.00 UTC works for me.

Bob, I previously adjusted the time from 15:00 UTC (7am your time) to 17:00 UTC (9am your time) per your request.
https://github.com/ethereumclassic/ECIPs/issues/460#issuecomment-1024583421

To verify, does this move to 9am (17:00 UTC) your time work for you?
Or do we need to move the meeting a second time to 11am (19:00 UTC) your time? @bobsummerwill 

Please confirm the desired time, I'd like to make sure this is the last change, if possible. Thanks.
FYI, 15:00 UTC is the most favorable time for the global participants. 19:00 UTC would be quite late for the Asian/Oceanic side of the world.

**Comment by bobsummerwill** [comment]
My bad.
I was confusing myself with the UTC to PST conversion.
The current time does work for me.
Thanks.

**Comment by gitr0n1n** [comment]
From @BelfordZ :

> I will never support a mining algorithm change, regardless of technical merits.

> I also refuse to spend more time above writing this comment on the matter. I have read all of the above discussion, reviewed each stated benefit and weakness, and thought long and hard about as many of the ramifications of this as possible. While each benefit on its own can be nitpicked over, having it's 'value added' objectively disseminated, there is 1.5 reasons that trump all benefit. It's an unfortunate reality of the world and humanity.

> The main point is that ruling out collusion being a driving force behind any contribution is impossible. This is especially true the closer the project gets to being connected with financial rewards. Every contribution has some level of Collusive Reward Potential. A change that adds a new opcode has a much higher CRP than fixing a documentation typo. Ranking changes with the highest CRP, my top 3 would be:

> Mining algorithm changes ('fair launch' being the oxymoron that we would be, for the 2nd time, subjected to)
> Consensus changes (blacklisting addresses, dependence on anything even remotely centralized for block validation)
Protocol defined Peering rules (ie drop a peer if they support protests in HK type of rules)
So, going back to the 1.5 reasons that trump all...

> 1 - To explain by counterposition, let's assume I was a large supporter of a mining algo change. What's to say I've not been paid by ASIC maker xyz to champion this change, giving them the jump on all other hardware providers?

> Spoiler: nothing.

> 0.5 - How can something which is designed to be inefficient be changed in the name of efficiency WITHOUT raising suspicion?

> Spoiler: it can't.

> To conclude, this might be a great proposal... for a new blockchain... And I urge you to reconsider this PR, as I believe there are more useful ways of spending development efforts.

https://github.com/ethereumclassic/ECIPs/issues/13#issuecomment-551256069

**Comment by antsankov** [comment]
I will attend the discussion on 2/21/22 (Monday next week) at 17:00 UTC time to give updates on the ECIP-1049 proposal. 

Also, Bob Summerwill is now a co-author of the proposal and its current champion, see https://github.com/ethereumclassic/ECIPs/issues/394#issuecomment-1041599611

**Comment by gitr0n1n** [comment]
> I will attend the discussion on 2/21/22 (Monday next week) at 17:00 UTC time to give updates on the ECIP-1049 proposal.
> 
> Also, Bob Summerwill is now a co-author of the proposal and its current champion, see [#394 (comment)](https://github.com/ethereumclassic/ECIPs/issues/394#issuecomment-1041599611)

Thanks for the response @antsankov . We will get that change of Champion on record on the CDC 22. You can expect a similar discussion format like the CDC 15, where everyone will have a chance to communicate. The goal here being to either get this proposal in a state where:
+ its proponents set a block and trigger a `contentious chain split` via a fork (since GPU miners will not support this proposal), or;
+ reject and the proponents of 1049 can stop hijacking every conversation and call, allowing the rest of the network participants to move forward building on top of a stable, mature PoW algo in Etchash.

See you Monday.

**Comment by DonaldMcIntyre** [comment]
@gitr0n1n you are conflating a technical ECIP process issue with your personal SHA3 change politics. This developer meeting was called to address a possibly stale ECIP (ECIP-1049) because you argued that 3 years had passed since its proposal, it is still in draft mode, that supposedly the champion did not make any changes or work on it all this time, and thus it had to be moved to accepted or rejected, where rejected is your preference because of your personal politics.

Regarding the ECIP technical issue, your argument that the ECIP has to be moved to rejected because the champion has not worked on it and that the community issues have not been addressed is false: 

- ECIP-1049 is the most currently debated change in the ETC ecosystem. 

- The new champion, Bob Summerwill, has not only been defacto champion of 1049 for several years, but he has invested money in development to advance it through the ETC Coop. 

- All developer teams have worked on it and even integrated it into their clients.

- All developer teams have tested it  on the ETC testnets. 

- There are plenty of videos and community calls about the SHA3 change in ETC that show that this is the most important and expected change in ETC.

- In fact, the last community call was precisely about ECIP-1049 and a proposal to add "hybrid proof of work": https://youtu.be/HQ9IKu3PVkA

In summary: ECIP-1049 not only alive but the most active ECIP in the ETC ecosystem. To move it to rejected as you propose would not only be wrong and unethical, but a violation of the ECIP process.

Regarding your personal politics about SHA3 and your opinions, you can keep on arguing on social channels. To use a ECIP process technicality is not the way to take down a proposal just because you don't like it.

Again, the politics of the SHA3 change belong in the debate forums, a call to reject ECIP-1049 has to be on the grounds of a clear rejection of it signaled from all stakeholders, but that is far from true.

**Comment by gitr0n1n** [comment]
> @gitr0n1n you are conflating a technical ECIP process issue with your personal SHA3 change politics. This developer meeting was called to address a possibly stale ECIP (ECIP-1049) because you argued that 3 years had passed since its proposal, it is still in draft mode, that supposedly the champion did not make any changes or work on it all this time, and thus it had to be moved to accepted or rejected, where rejected is your preference because of your personal politics.
> 
> Regarding the ECIP technical issue, your argument that the ECIP has to be moved to rejected because the champion has not worked on it and that the community issues have not been addressed is false:
> 
> * ECIP-1049 is the most currently debated change in the ETC ecosystem.
> * The new champion, Bob Summerwill, has not only been defacto champion of 1049 for several years, but he has invested money in development to advance it through the ETC Coop.
> * All developer teams have worked on it and even integrated it into their clients.
> * All developer teams have tested it  on the ETC testnets.
> * There are plenty of videos and community calls about the SHA3 change in ETC that show that this is the most important and expected change in ETC.
> * In fact, the last community call was precisely about ECIP-1049 and a proposal to add "hybrid proof of work": https://youtu.be/HQ9IKu3PVkA
> 
> In summary: ECIP-1049 is not only alive but the most active ECIP in the ETC ecosystem. To move it to rejected as you propose would not only be wrong and unethical, but a violation of the ECIP process.
> 
> Regarding your personal politics about SHA3 and your opinions, you can keep on arguing on social channels. To use an ECIP process technicality is not the way to take down a proposal just because you don't like it.
> 
> Again, the politics of the SHA3 change belong in the debate forums, a call to reject ECIP-1049 has to be on the grounds of a clear rejection of it signaled from all stakeholders, but that is far from true.

Please note, prior to this CDC 22 meeting being called- the proposal was abandoned after three years. The fact that the proponents are now recognizing they have not been following the ECIP does not negate the need for the call.

It's great the @bobsummerwill has volunteered to act as the champion for this abandoned proposal. But that only happened after this meeting was put on the docked.

I have not concealed that I am personally in opposition of ECIP-1049 becuase i believe its a very `bad idea` from its foundation and has violates the ECIP process at many points in its existence, including the proponents attempt to push this incomplete and vague proposal to `Accepted` status in Septemeber 2020- which triggered the CDC 15 call.

However, none of those personal opinions relate to the proponents failure to follow the ECIP process at many different stages. As cited, there are numerous reasons this proposal is in violation of the ECIP process- an abandoned proposal with no champion was just ONE of many reasons.

**Comment by gitr0n1n** [comment]
Note, the formal PR to move ECIP-1049 to `Rejected` status has been drafted:
https://github.com/ethereumclassic/ECIPs/pull/465

**Comment by ETCpunk2** [comment]
@gitr0n1n  I was able to grab this before you deleted it the other night in discord, so that we can discuss its full implications with the community present. 

**Call to Action:**
Ronin has threatened the network and community multiple times over the last few months, attempting to hold it hostage to his will, which is to maintain the current etchash algorithm. While he himself may not be a "bad actor" as he calls others, his actions are that of a terrorist conducting large scale extortion. My motion is that he should cease making open threats against the community, should maintain an willingness to cooperate on whatever community decisions get made, or else his role as an ECIP Editor be revoked immediately and that he be ejected from the community social channels. 

**Evidence:**
![ronin_backdoor_meeting](https://user-images.githubusercontent.com/100152358/154997505-cdcbc780-5b9f-4140-bb64-c21c600af77b.jpg)
<img width="711" alt="bad actors ronin" src="https://user-images.githubusercontent.com/100152358/154997691-a44a7e1e-4a69-4d26-9f51-9df493403cd4.png">


**Point #1** 
You have previously claimed that ETC is a totally decentralized network, and yet in the same chat room you now express (see picture below) concern about the Coop being the only development group in the community. The timing and contradictions in your actions are very confusing and you must clarify them. 

- Is there a reason why you weren't addressing this concern at any point in the last 12 months, since the withdrawal/dissolution of Core and Labs? What about when IOHK re-entered the community and you/pistov/et al successfully drove them back out, leaving Coop on their own again? 
- What is it about this proposal that has you so worried about having a decentralized development community that you suddenly, without discussing it in the community, arrange a backdoor contingency of private and unknown developers who, apparently are instructed to support the network ONLY if this proposal goes through? 
see this comment from you: https://discord.com/channels/223674353001168906/750122538629070888/944800521997406299 

**My thoughts on Point #1:** 
It surely can't be Bob that concerns you, because Bob has been in charge of the Coop as the only active dev group remaining in the ecosystem for over a year (I don't count IOHK as they were only engaged here for a few months before withdrawing). The Coop has been the only group maintaining ETC client software for quite some time now. Throughout a window of time in which the ETC developers have been about as centralized as you can possibly get, Bob, as the leader of the Coop, has shepherded his power and authority with humility, open-mindedness, and transparency. They have been responsible for two successful hardfork upgrades in that period, they put out thorough quarterly reports, and they communicate frequently with informative monthly newsletters. 

I don't see a single piece of meaningful evidence that Bob and Coop have bad intentions, underhanded dealings, or malevolence towards ETC (including toward you). You have had over a year to communicate your concern and put together an action plan. I don't feel this is necessary to make my point, but I'm pretty sure I could look through discord and GitHub and find that you were VERY active in pushing IOHK out of the community. 

My opinion on the contradictions in your newly disclosed actions to procure a "backup" dev team to assist you in maintaining the network in the event you were unsuccessful in stopping ECIP-1049 is that by you doing said actions, you have now demonstrated that you are in fact vying for centralized control of ETC. If you weren't, you would have been pushing to start a secondary and tertiary dev group this whole time. But clearly it is only this specific proposal that does not suit your plans; you are unable to pressure Coop into compliance, and have now gone to extremes (in terms of contradicting yourself), to get your way. It is pure hypocrisy. 

**Point #2** 
I have pointed out previously that ETC is currently extremely centralized in hashrate, pools, developers, and even the list of exchanges. I am in full support of creating and/or inviting more dev groups into the ecosystem as that would be a boon for the network. However, it goes without saying that if a dev group wants a warm welcome, there should be an open conversation with the community on the matter well ahead of time. 

- Who are these developers? 
- What is their motivation for working on ETC? 
- What segment(s) of the ecosystem are they interested in? Core, Defi, Meta, NFTs? 
- What if ECIP-1049 is approved? To what degree will that affect their interest? 
- What if ECIP-1049 is denied? To what degree will that affect their interest? 
- Funding source? 
- Are their team members self-doxxed like many of the other great devs in ETC were? 
- Would they join a community call to field questions? 

**My thoughts on Point #2:** 
If this dev team is only here to act as your personal "guard dogs" against ECIP-1049, then I will not support their participation in the community or on ETC. But if they are willing to answer those questions above, speak to us on a community call about their *interesting* and timely interest in ETC, and most importantly, they are willing to stick around regardless of the outcome ECIP-1049, then it may in fact be a good idea to get them involved here. 

**Summary:** 
Since you have now publicly announced a back door agreement with an anonymous dev team that will provide "redundancy" to the network, only if 1049 gets approved, and you have procured this arrangement in secret without transparency to the community, it is clearly unacceptable behavior that goes well outside the bounds of the community guidelines and 

_**-- I motion that further efforts to hold the community hostage should be returned with an immediate removal of your ECIP Editor status and ejection from community channels.  --**_ 

You are behaving contrary to the Founding Principles on decentralization that you claim to embody so much. Your call on today is unsubstantiated, divisive, a waste of all our time, and avenue for you to weaponize your status as ECIP Editor. If this dev team that you have procured was honest, then they would introduce themselves and face the music of inquiry; and you certainly wouldn't attempt to use them as leverage against the Coop (the very folks who have been working hard to keep this chain going amidst a relentless onslaught of challenges) and ECIP-1049 proponents. 

**Re: "redudancy"**
In discord @gitr0n1n claims that I am against redundancy. I am not. That's the point of decentralization, to have so many point of redundancy that the network can't fail or be broken.

I am posting my response to him here to the conversation congruent, as well as to address that false logic:
![redundancy](https://user-images.githubusercontent.com/100152358/154998897-de70bf75-5459-4df5-9228-d7a82019265e.png)

**Comment by gitr0n1n** [comment]
Call Results:

Productive Topics from this call:
+ A new ECIP Champion was selected in @bobsummerwill , @antsankov has stepped down from the proposal as he is too busy and inactive.
+ Recommendations to split the ECIP to multiple ECIPs to align with e ECIP-1000; technical work, transition strategy, activation plan.
+ The elephant in the room on this one has been "consensus" as the proposal aims to displace the current mining ecosystem. It's unlikely ETChash miners will update nodes to this proposal. Pushing contentious hard forks on the network is a violation of the ECIP process.

Given the new champion is active, @bobsummerwill is provided time to update the 1049 proposal to be ECIP-1000 compliant. This was recommended in Sept 2020 to the previous champion after CDC 15, but did not happen. The hope is with a new champion we will see compliance with the ECIP process.

Some comments from @bobsummerwill on the undocumented technical work from his staff:
+ Support for Astor already integrated in Besu.   There is a pull request for core-Geth (and blocks successfully mined on Astor), but some rearchitecting needed to make that clean enough to integrate.

+ The transition is where more client work is needed (ie. having a Testnet which goes through the transition versus being Keccak256 from Genesis).   More work needed on mining side software too.

+ So not tons left to do, but probably needs a couple of months of focus to do it right.

+ Note - this is the technical work.  Roll out via testnets, all the social layer etc etc is more time again, of course.

+ I was asked where June (2022) was feasible (end to end).  I think that is a pretty clear not really possible.

Video of the call: https://www.youtube.com/watch?v=SV3WJwEfJwo

Here is the open PR to move 1049 to `Rejected` status if the new champion does not bring the proposal up to ECIP-1000 standards:
https://github.com/ethereumclassic/ECIPs/pull/465

Thanks for everyone's participation.
